tests/ui/dis/issue-723.rs

```diff
@@ -0,0 +1,11 @@
+// build-pass
+// compile-flags: -Ctarget-feature=+RayTracingKHR,+ext:SPV_KHR_ray_tracing
+// compile-flags: -C llvm-args=--disassemble-globals
+// normalize-stderr-test "OpCapability VulkanMemoryModel\n" -> ""
+// normalize-stderr-test "OpExtension .SPV_KHR_vulkan_memory_model.\n" -> ""
+// normalize-stderr-test "OpMemoryModel Logical Vulkan" -> "OpMemoryModel Logical Simple"
+
+use spirv_std as _;
+
+#[spirv(closest_hit)]
+pub fn test_closest_hit(#[spirv(object_to_world)] _object_to_world: [glam::Vec4; 3]) {}
```

Comment:
Probably want to use a slightly modified test case, `object_to_world` requires an OpTypeMatrix on vulkan and so it's going to give an error when compiletest has target env of vulkan.

Reply:
Ah huh I haven't been running `cargo compiletest` with any flags lately because U thought Vulkan was the default.